Sources/LiveViewNative/Views/Controls and Indicators/Value Inputs/Stepper.swift

```diff
@@ -66,6 +67,7 @@ struct Stepper<R: RootRegistry>: View {
     @Attribute("upper-bound") private var upperBound: Double?
     
     public var body: some View {
+        #if !os(tvOS)
```

Comment:
Hmm, if you have `@available(iOS 13.0, macOS 10.15, watchOS 9.0, *)` above on the struct `Stepper` do you need this `#fi !os(tvOS)`?

Reply:
The `#if` excludes it at compile time to prevent build errors.